src/pages/Version/get-route-package-name.ts

```diff
@@ -0,0 +1,9 @@
+function getRouterPackageName(packageName: string, scope?: string): string {
```

Comment:
Move this to utilities in a new PR, also router is misleading, it just build a package name based on scope. A `getPackageName` would fit better here.